quickstart/docker-compose.yml

```diff
@@ -98,6 +98,54 @@ services:
       - 9090
     volumes:
       - ../quickstart/exampledata/config/prometheus/prometheus.yml:/opt/bitnami/prometheus/conf/prometheus.yml
-
+  config:
+    image: nginx:latest
+    container_name: config
+    profiles:
+      - basic_auth
+    network_mode: host
+    expose:
+      - 8081
+    volumes:
+      - ../quickstart/exampledata:/usr/share/nginx/html:ro
+      - ../quickstart/exampledata/config/nginx/nginx.conf:/etc/nginx/nginx.conf:ro
+      - ../quickstart/exampledata/config/nginx/conf.d:/etc/nginx/conf.d:ro
+  keycloak:
+    image: bitnami/keycloak:latest
+    container_name: keycloak
+    network_mode: host
+    expose:
+      - 8080
+    profiles:
+      - oauth2
+    environment:
+      KEYCLOAK_ADMIN_USER: admin
+      KEYCLOAK_ADMIN_PASSWORD: admin
+      KEYCLOAK_DATABASE_HOST: localhost
+      KEYCLOAK_DATABASE_PORT: 5432
+      KEYCLOAK_DATABASE_NAME: keycloak
+      KEYCLOAK_DATABASE_USER: keycloak
+      KEYCLOAK_DATABASE_PASSWORD: bitnami
+      KEYCLOAK_HTTP_PORT: 8080
+      KEYCLOAK_HTTPS_PORT: 8443
+      KEYCLOAK_BIND_ADDRESS: 127.0.0.1
+  keycloak_db:
+    image: bitnami/postgresql:latest
+    container_name: keycloak_db
+    network_mode: host
+    expose:
+      - 5432
+    profiles:
+      - oauth2
+    environment:
+      POSTGRESQL_PASSWORD: bitnami
+      POSTGRESQL_POSTGRES_PASSWORD: bitnami
+      POSTGRESQL_DATABASE: keycloak
+      POSTGRESQL_USERNAME: keycloak
+    volumes:
+      # use this folder to persist your postgresql data by dumping the database to it after changes
+      # outside the container: chmod 777 quickstart/exampledata/config/postgresql
+      # inside the container: pg_dump keycloak -U keycloak -W --file /docker-entrypoint-initdb.d/keycloak_db.sql
```

Comment:
I tried that one here but I get a `password authentication failed for user "keycloak"`, I tried the password `bitnami` and `keycloak` but both are not working. Which one do I have to use.